streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/camera_point_add.html

```diff
@@ -66,9 +66,6 @@
 
 <div class="container">
     {% if user.is_authenticated %}
-        {% if added %}
-        <strong>{% trans "You have successfully added a new camera point." %}</strong>
```

Comment:
should this code be moved to the `camera_point_view` template?
I feel like it would be useful there